lib/api.js

```diff
@@ -87,11 +87,13 @@ function FHapi(cfg, logr) {
 }
 
 /**
- * Initilisation returns the $fh object to clients
+ * Initialisation returns the $fh object to clients
  */
 module.exports = (function () {
-  // First setup the required config params from  env variables
+  // First setup the required config params from env variables
   var millicore = process.env.FH_MILLICORE || 'NO-MILLICORE-DEFINED';
+  var millicore_protocol = process.env.FH_MILLICORE_PROTOCOL || 'https';
```

Comment:
If these are new env vars we should document them somewhere, probably in the README.